src/Core/Content/Product/SalesChannel/SalesChannelProductEntity.php

```diff
@@ -15,71 +15,27 @@
 #[Package('inventory')]
 class SalesChannelProductEntity extends ProductEntity
 {
-    /**
-     * @var PriceCollection
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $calculatedPrices;
+    protected PriceCollection $calculatedPrices;
 
-    /**
-     * @var CalculatedPrice
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $calculatedPrice;
+    protected CalculatedPrice $calculatedPrice;
 
-    /**
-     * @var PropertyGroupCollection|null
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $sortedProperties;
+    protected ?PropertyGroupCollection $sortedProperties = null;
 
-    /**
-     * @var CalculatedCheapestPrice
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $calculatedCheapestPrice;
+    protected CalculatedCheapestPrice $calculatedCheapestPrice;
 
-    /**
-     * @var bool
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $isNew = false;
+    protected bool $isNew = false;
 
-    /**
-     * @var int
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $calculatedMaxPurchase;
+    protected int $calculatedMaxPurchase;
 
-    /**
-     * @var CategoryEntity|null
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $seoCategory;
+    protected ?CategoryEntity $seoCategory = null;
 
     /**
      * The container will be resolved on product.loaded event and
      * the detected cheapest price will be set for the current context rules
-     *
-     * @var CheapestPrice|CheapestPriceContainer|null
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
      */
-    protected $cheapestPrice;
+    protected CheapestPrice|CheapestPriceContainer|null $cheapestPrice = null;
```

Comment:
Wonder if it should be like this instead?

```suggestion
    protected CheapestPrice|null $cheapestPrice = null;
```

But from the phpdoc annotation, it might be CheapestPriceContainer as well, so I keep it the same for safety reasons. Do you know if it's a typo @shopware/product-ct-core  🙈

Reply:
yes, seems to be possible, that it contains the container: https://github.com/shopware/shopware/blob/trunk/src/Core/Content/Product/Subscriber/ProductSubscriber.php#L63